src/request.lisp

```diff
@@ -74,6 +74,9 @@
                       append (quri:url-decode-params kv :lenient t)))
           (rplacd (last env) (list :cookies (request-cookies req))))))
 
+    (let ((a 2))
```

Comment:
<sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
style-warning: The variable A is defined but never used.